athena/tests/modules/text/module_text_llm/mock/test_default_approach_mock.py

```
@@ -1,17 +1,33 @@
 from module_text_llm.default_approach.generate_suggestions import generate_suggestions
+from module_text_llm.default_approach.schemas import FeedbackType
 from athena.text import Exercise, Submission, Feedback
 from athena.schemas.exercise_type import ExerciseType
-from modules.text.module_text_llm.mock.utils.mock_env import mock_sent_tokenize
-from modules.text.module_text_llm.mock.utils.mock_llm import MockAssessmentModel
-from modules.text.utils.mock_llm import MockFeedbackModel
+from modules.text.module_text_llm.mock.utils.mock_llm import MockAssessmentModel, MockFeedbackModel
 from athena.schemas.text_submission import TextLanguageEnum
 
 
+class MockFeedbackModel(BaseModel):
```

Ruff (0.13.3)

Redefinition of unused `MockFeedbackModel` from line 9

(F811)